### ISSUE TITLE
[RFC][Mod]Add The Aether I

### ISSUE THREAD

**Issue by bartimaeusnek** [GTNewHorizons/GT-New-Horizons-Modpack#7301]
#### What do you suggest instead/what changes do you propose?

I would like to add "The Aether I" to the pack, since it now has an updated 1.7.10 version which is GPLv3 Licensed, i dont see a reason why we shouldnt add it. It would bring a lot of exploration to the pack and a quite unusual dimension which can be explored, also its def. more intresting than any of the planet dims, compareatively.

https://www.curseforge.com/minecraft/mc-mods/the-aether

https://github.com/Gilded-Games/The-Aether-Archived

**Comment by Ethryan** [comment]
Forked it for preliminary testing.

**Comment by bartimaeusnek** [comment]
> bedrill
T1 rocket gated behind aether
we do not have HV dimension
well, besides useless last milenium i guess
add aluminum ore vein, make titanium required for t1 rocket, since you can get titanium from bauxite
We could go even further and gate aether behind stainless steel level wand, requiring 50/75 vis to craft portal, but i guess a lot of people do not like magic
but that would solve HV gate for this dimension

**Comment by dvdmandt** [comment]
At the moment, magic is mostly an optional yet very useful side quest, and with the extremely punishing warp effects that can't really be countered until much later, I think we should keep it that way.

What if instead of gating main progression behind the Aether, we gated basically all of EnderIO behind it? It's like semi-magical, not really required yet extremely useful..?

**Comment by leagris** [comment]
I vote for players who already reached a given tier are excluded from voicing any opinion on nerfing lower tiers.

**Comment by boubou19** [comment]
From a pack perspective: 

- it adds new mobs, so who knows what weird interections will happen with infernal mobs/special mobs.
- It is a new mod to fork and maintain. 

From a gameplay perspective:
- what will we put in this dimension to make it interesting? As bedrill pointed out on discord, we don't have HV dimension but do we really need one?
- By the time you are HV, you already have powerful weapons. So any boss battle will be boring.

From an item perspective, Some items seems to be interesting:
- invisibility cloak could be an alternative to the OP the one ring
- the iron bubble to breathe under water
- vampire Blade that applies life steal when hitting enemies
- Life Shard that adds one permanent full heart-container
- the golden feather that prevents fall damage
- the regeneration stone that regenerates health over time, could be a good alternative to the healing axe / the "eat food and wait to regen" mecanic to get some regen
- the shield of repulsion that put a distance between the player and the enemies (useful to recover stuff if you died in a mob pit)
- the pig slayer, 1HKO pigs and zombie pigmens, useful when mining in the nether
- the holy sword, that can 1HKO the undead mobs (seems a bit OP tho)
- some armor equally better than diamond armor or stronger, with some perks

this list is not exhaustive, but it can be a good start to reason around it. However, i feel like many stuff of that list can has an equivalent availiable before HV. Unless we add worthy content from the progression inside this dimension, there is no need to gate it behind HV imo. I would rather see a way to get Aether access at late LV given the fact that most of the stuff from the Aether feels "required" in this tier, otherwise, the dim will just be avoided like many magical stuff in the pack.

**Comment by dvdmandt** [comment]
On one hand, I really don't think we need to nerf early tiers or change main progression, but on the other hand I would really like to see more exploration and combat content. Currently, apart from the mostly symbolic GC dungeons and bosses, dungeon crawling is basically optional but gives useful and desirable loot and unlocks. I think that's pretty good, and I think we could do something similar with the Aether. 

I think EnderIO is a pretty decent candidate since it fits very well with the useful but not mandatory thing. Perhaps there are some multiblocks that are useful but not required, and/or slightly overpowered? Perhaps we don't need to hard gate anything, just make it significantly easier to produce after progressing in the Aether?

As for access to powerful weapons making any boss fights boring - we would definitely need to tweak any mobs and bosses to be balanced around what you are expected to have access to at the time you get there.

**Comment by bedrill64** [comment]
2/3 Bosses in Aether 1 have their mechanic not dependent on the weapon power, Slider has to be damaged with pickaxe, and strength potion doesn't increase pickaxe dmg, and Sun Spirit requires attacking him with these weird ice balls, only valkyrie queen can be one shot with crossbow I think.

**Comment by KiloJoel** [comment]
If it is integrated into progression, maybe MV could be a nice place to put it? Given it's a fairly small tier currently

**Comment by bedrill64** [comment]
> If it is integrated into progression, maybe MV could be a nice place to put it? Given it's a fairly small tier currently

I'm fine with MV, still, aluminum ore vein would be awesome for the tier, as an additional direct source. PLUS Titanium source for T1 rocket :yobaface:

**Comment by bedrill64** [comment]
And if we were to gate EIO behind Aether we could add some kind of Tantalum vein with higher weight than OW and Nether to the dimension.

**Comment by Vernam** [comment]
lol, we already have Dim in Hv - End. Let's l don't forget about it. And i don't think that we need new dim for Mv tier. On this tier we have a lot of quests and work to do since mv gated first magic. Aether will not add interesting gameplay, just add some meaningless slowness for progression. Maybe try to add real interesting gameplay without adding tons of Dims everytime?) Who can say really interesting argument for adding new "adventure" dim ?) And suggestion to move Bauxite to Aether and do T1 rocket with titanium - this brakes any reason to fly on the moon. Add new Dim to break Moon dim?) If u want to add Aether u need to imagine something exclusive without add slowness of progression and breaking logic.

**Comment by Prometheus0000** [comment]
What about adding it as a GC planet instead? For more interesting terrain at least. If it's possible.

**Comment by General-Ike** [comment]
I would love to see the addition of The Aether I. My opinion is that it should be an auxiliary, magic locked dimension, accessible in MV. Off the top of my head, I'd think the portal could be made in some way involving Angel Blocks and activated with some thaumic fluid like purifying liquid (trying not to get bogged down with details). IE: needs a wand and an amount of research to make it take some effort.

What to gain: I personally like the idea of a magic focused dimension, cause magic could use some love in this pack. As it is an "angelic" realm, maybe it could give higher tier warp eliminators for defeating the bosses, making it invaluable to thaumaturgists, and something uniquely, and thematically, available in the Aether. For resources, I like the concept of granting easier/earlier access to something via the Aether (titanium/aluminium are good where they are though, no bypassing the moon's relevance). Tantalite is a convincing suggestion. I'd also like to see amethyst, giving another route to make prasiolite, also making the Aether relevant at a later tier.

Hot take (why won't my tech work?): I've found GT options constantly overshadow the non-tech options in the modpack. What if the Aether interfered with tech, rendering it useless? Examples:
-Any energy containing block/item is immediately (or rapidly) drained in the Aether.
-Tinker's tools are considered "ineffective" against all blocks and deal 1 dmg to all creatures (like vamp pigment)
Rather than balancing the Aether around the tech options available, you require branching out into other options, even at high tiers. Balancing bosses also becomes substantially easier.

**Comment by Vernam** [comment]
> I would love to see the addition of The Aether I. My opinion is that it should be an auxiliary, magic locked dimension, accessible in MV. Off the top of my head, I'd think the portal could be made in some way involving Angel Blocks and activated with some thaumic fluid like purifying liquid (trying not to get bogged down with details). IE: needs a wand and an amount of research to make it take some effort.
> 
> What to gain: I personally like the idea of a magic focused dimension, cause magic could use some love in this pack. As it is an "angelic" realm, maybe it could give higher tier warp eliminators for defeating the bosses, making it invaluable to thaumaturgists, and something uniquely, and thematically, available in the Aether. For resources, I like the concept of granting easier/earlier access to something via the Aether (titanium/aluminium are good where they are though, no bypassing the moon's relevance). Tantalite is a convincing suggestion. I'd also like to see amethyst, giving another route to make prasiolite, also making the Aether relevant at a later tier.
> 
> Hot take (why won't my tech work?): I've found GT options constantly overshadow the non-tech options in the modpack. What if the Aether interfered with tech, rendering it useless? Examples:
> -Any energy containing block/item is immediately (or rapidly) drained in the Aether.
> -Tinker's tools are considered "ineffective" against all blocks and deal 1 dmg to all creatures (like vamp pigment)
> Rather than balancing the Aether around the tech options available, you require branching out into other options, even at high tiers. Balancing bosses also becomes substantially easier.

well... What do u mean when saying "Rather than balancing the Aether around the tech options available, you require branching out into other options, even at high tiers. Balancing bosses also becomes substantially easier." It is GREGTECH oriented modpack! it's logical that everything is balancing around tech options. What could give u Aether that approves balancing tech around Aether?) That can give boss drops to original gameplay that justifises forcing tham to kill?) 

Suggest with tantal - the best suggestion, but it don't worth  to add mod.

Maybe we need to think more creatively? Maybe add on this dim new bedrock fluids that will properly motivate any player to  explore this Dim? Maybe, like with space rocks - add  processing to Aether rocks with interesting products that wil worth it. Maybe add same thaum veins as in TF, but fith another dense of shards ( less hards, more Cinnabar) or add Infused gold on this vein ( that will approve needs for gold, salis mundus, mercury for ore processing). For Trees from Dim add more powerful recipes to Pyrolyse oven and Biomass. For mobs - add useful drop.  And for lategame? maybe, add that Air intake hatch will collect Helium instead of air ( or nooble gas, i'm just imagine )
Without all of this adding new dim is not approved. We need Content, content for magical/non magical players. 

Now i do not see reason to kill Aether bosses - this is big problem. In TF killing the bosses opens more space of Dim for digging veins. Although  i just thinked that bosses could drop some stuff like lenses that will boost come recipes. But it needed to be thinked more than a few minutes :D

**Comment by Impos913** [comment]
@Vernam The End isn't HV lol. It is stone age. Ender eyes aren't gated. Also you can already skip rockets. You don't need T1 rocket for constant and large amounts of titanium or T2 rocket for tungsten.

**Comment by General-Ike** [comment]
When I refer to branching out, I mean that the magic equivalent option to tech items is normally inferior at most points of the game (or at the very least, not worth the effort) I'm trying to give incentive to not just make a nanosuit and call it a day. Since magic progression is already gated by GT progression, this doesn't encourage bypassing tech, it encourages exploration of non-tech options once you unlock them (so you can't just be an ZPM tier player and say "I'll just drop by the Aether with my gravisuite armor to grab 'x' resource" and ignore any real features of the dimension. As for "balance" I'm referring to not being able to one-shot the Valkyrie Queen with my 272 dmg, 4 shots per second tinkers crossbow.

Also, thinking as a magic dimension, I suggested warp eliminating/reducing items as drops from bosses as a uniquely powerful item that is of serious help to pre-LuV/IV thaumaturgists. Not that that would be the *only* drop, but I guarantee, that *alone* would make the dimension worth exploring for anyone doing magic.

**Comment by GTNH-Colen** [comment]
Aether is a mod I personally adore so I would like to see it added but am concerned again over the size of the pack, will need to see how this plays out in that regard.

I can see this being integrated into magic progression, I would definitely want to see it intertwined into the rest of the pack and not just added for the sakes of it. It's a much more interesting dimension than the twilight forest that people have seen countless times by this point. Could even be a perfect place to add requirements for botania if we ever get around to adding that as well ;)

**Comment by Dream-Master** [comment]
I don't get it. I want to add Logistic pipes and every people cry tps goes down all got laggy and the pack loads much slower. Now you want aether mod and everyone say yes let add it. not much concerns about laggy servers.

**Comment by Vernam** [comment]
> @Vernam The End isn't HV lol. It is stone age. Ender eyes aren't gated. Also you can already skip rockets. You don't need T1 rocket for constant and large amounts of titanium or T2 rocket for tungsten.

this is abuse and do not thinked like that. ofcourse there are some stupid options ti skip this, but u still need rocket to progress the space travel.  end is conceived as an hv dimension, even though there are some ways to get an ender eye (but I haven't seen it in the inhabitants, there is still selection, which takes an extremely long time to reach the enderbloom), however, the dimension requires flight, and this is the mv-hv level. Quests for the End open on the HV tier, so it is not necessary to say that it is not Hv tier. It's meant to be that way.

**Comment by Vernam** [comment]
> When I refer to branching out, I mean that the magic equivalent option to tech items is normally inferior at most points of the game (or at the very least, not worth the effort) I'm trying to give incentive to not just make a nanosuit and call it a day. Since magic progression is already gated by GT progression, this doesn't encourage bypassing tech, it encourages exploration of non-tech options once you unlock them (so you can't just be an ZPM tier player and say "I'll just drop by the Aether with my gravisuite armor to grab 'x' resource" and ignore any real features of the dimension. As for "balance" I'm referring to not being able to one-shot the Valkyrie Queen with my 272 dmg, 4 shots per second tinkers crossbow.
> 
> Also, thinking as a magic dimension, I suggested warp eliminating/reducing items as drops from bosses as a uniquely powerful item that is of serious help to pre-LuV/IV thaumaturgists. Not that that would be the _only_ drop, but I guarantee, that _alone_ would make the dimension worth exploring for anyone doing magic.

end is conceived as an hv dimension, even though there are some ways to get an ender eye (but I haven't seen it in the inhabitants, there is still selection, which takes an extremely long time to reach the enderbloom), however, the dimension requires flight, and this is the mv-hv level. Quests on the End open on the HV tier, so it is not necessary to say that it is not Hv tier. It's meant to be that way)

**Comment by Vernam** [comment]
All the ideas described above for implementing mod extremely strongly affect the already existing modpack logic, for "adventure" mod this is too much work, which does not justify itself. 

Before adding new mods for the development of modpack, we should focus and solve the problems that already exist, and not fill modpack with a bunch of mods that duplicate the content and that take a lot of time to logically implement them.

**Comment by GTNH-Colen** [comment]
> I don't get it. I want to add Logistic pipes and every people cry tps goes down all got laggy and the pack loads much slower. Now you want aether mod and everyone say yes let add it. not much concerns about laggy servers.

One thing to keep in mind is that everything in logistic pipes is already done by like 3 other mods so it's kinda redundant, the aether is unique and honestly not in enough packs.

**Comment by boubou19** [comment]
> Before adding new mods for the development of modpack, we should focus and solve the problems that already exist, and not fill modpack with a bunch of mods that duplicate the content and that take a lot of time to logically implement them.

considering the fact that the pack is still in dev since litterally years, some stuff will take a long time to be adressed/reworked (such as UV+ content for exemple). Adding a mod now, when some stuff is not finished/ having issues is easier to integrate than integrate it once everything is done.

**Comment by Vernam** [comment]
> > I don't get it. I want to add Logistic pipes and every people cry tps goes down all got laggy and the pack loads much slower. Now you want aether mod and everyone say yes let add it. not much concerns about laggy servers.
> 
> One thing to keep in mind is that everything in logistic pipes is already done by like 3 other mods so it's kinda redundant, the aether is unique and honestly not in enough packs.

that unique in the Aether? same as TF add Dim and Bosses, some decorating staff and thighs. i don't see alot of "unique" :D




> > Before adding new mods for the development of modpack, we should focus and solve the problems that already exist, and not fill modpack with a bunch of mods that duplicate the content and that take a lot of time to logically implement them.
> 
> considering the fact that the pack is still in dev since litterally years, some stuff will take a long time to be adressed/reworked (such as UV+ content for exemple). Adding a mod now, when some stuff is not finished/ having issues is easier to integrate than integrate it once everything is done.

But most of suggestions is to add Aether to early-midgame wich enough reworked already and i do not think that this is a good idea to rework mid-earlygame again. In a few months later someone will add a new mod-suggestion with same argument - and it will repeat again and again and again...  I don't think we should indulge every little whim of every player because otherwise modpack will be in development forever. In the case of logical changes-yes, but not in the case of adding duplicate content, which will take a lot of time and effort instead of this time spent on development. 

So far, I have not seen any arguments in favor of this mod. A lot of love for mod, not a single weighty argument. What unique content will the mod add? Another boss? Maybe the mod has a unique crafting system? which would fit perfectly into the theme of the build? I don't remember that...

**Comment by boubou19** [comment]
> But most of suggestions is to add Aether to early-midgame wich enough reworked already and i do not think that this is a good idea to rework mid-earlygame again

this part of the game is the played the most, so it's normal to have a lot of feedback on it and change it often. Also, HV is barely mid-game.

> A lot of love for mod, not a single weighty argument.

Your statement is purely subjective. Aether I adds one dimension to explore and some new bosses, 2 of them having a combat not based on weapon strengh. Items added by Aether I have alternatives in the pack already. While it can be considered as partial bloat, it can also be considered as alternatives. And in GT:NH, it's a philosophy to have choice, so player can do what he wants, without being forced to follow one way to obtain the wanted item/ability.

The reason why LP was not well received is because it is too similar with project red: most of the items in LP have an equivalent in PR. Aether I items have equivalents, but not exactly the same, so it's somewhat unique while being similar to different stuff.

**Comment by Vernam** [comment]
> > Your statement is purely subjective. Aether I adds one dimension to explore and some new bosses, 2 of them having a combat not based on weapon strengh. Items added by Aether I have alternatives in the pack already. While it can be considered as partial bloat, it can also be considered as alternatives. And in GT:NH, it's a philosophy to have choice, so player can do what he wants, without being forced to follow one way to obtain the wanted item/ability.
> 

this argument can even justify the presence of project E in modpack. Mod content is quite small to compete in terms of variability

> The reason why LP was not well received is because it is too similar with project red: most of the items in LP have an equivalent in PR. Aether I items have equivalents, but not exactly the same, so it's somewhat unique while being similar to different stuff.

well, But wuth Aether the same thing! please be more specific  about "not exactly the same, so it's somewhat unique while being similar to different stuff". We need some examples... All what wrote earlier has full equivalence and seemed not quite enough reason to be added.. It needed to be discuss, i think =)

**Comment by Vernam** [comment]
> > > Your statement is purely subjective. Aether I adds one dimension to explore and some new bosses, 2 of them having a combat not based on weapon strengh. Items added by Aether I have alternatives in the pack already. While it can be considered as partial bloat, it can also be considered as alternatives. And in GT:NH, it's a philosophy to have choice, so player can do what he wants, without being forced to follow one way to obtain the wanted item/ability.
> 
> this argument can even justify the presence of project E in modpack. Mod content is quite small to compete in terms of variability
> 
> > The reason why LP was not well received is because it is too similar with project red: most of the items in LP have an equivalent in PR. Aether I items have equivalents, but not exactly the same, so it's somewhat unique while being similar to different stuff.
> 
> well, But with Aether the same thing! please be more specific about "not exactly the same, so it's somewhat unique while being similar to different stuff". We need some examples... All what wrote earlier has full equivalence and seemed not quite enough reason to be added.. It needed to be discuss, i think =)

**Comment by 0lafe** [comment]
you don't think we have enough useless exploration dims? Shoutout the TF/nether/end/planets/overworld lol

I can't hate because the aether is cool and all, but im not sure it's cool enough I'd ever want to actually go there. Hot take, im not sure exploration is that great of a mechanic. Sure you can see some cool terrain gen, but flying/gliding around for x mob/item/structure/ore imo is incredibly boring, and the aether is no different. Imo pre rocket doesn't really need more visually different dims, and "exploration" and such is incredibly tedious post rockets. 

Maybe we can swap this for the TF to gate thaum? Lowkey hate the majority of the TF, fuck mazes

**Comment by jackowski626** [comment]
> On one hand, I really don't think we need to nerf early tiers or change main progression, but on the other hand I would really like to see more exploration and combat content. Currently, apart from the mostly symbolic GC dungeons and bosses, dungeon crawling is basically optional but gives useful and desirable loot and unlocks. I think that's pretty good, and I think we could do something similar with the Aether.
> 
> I think EnderIO is a pretty decent candidate since it fits very well with the useful but not mandatory thing. Perhaps there are some multiblocks that are useful but not required, and/or slightly overpowered? Perhaps we don't need to hard gate anything, just make it significantly easier to produce after progressing in the Aether?
> 
> As for access to powerful weapons making any boss fights boring - we would definitely need to tweak any mobs and bosses to be balanced around what you are expected to have access to at the time you get there.

dungeon crawling is lame in gtnh tbh. there is what? roguelike dungeons only iirc. they are boooring. I'd populate the aether (and/or other stuff) with better dungeon dungeons

**Comment by DbpGaming** [comment]
> 
> 
> I don't get it. I want to add Logistic pipes and every people cry tps goes down all got laggy and the pack loads much slower. Now you want aether mod and everyone say yes let add it. not much concerns about laggy servers.

i mean aether 1 and logistic pipes relatively be negligible in TPS drops, compared to some of the other stuff the pack has like tons of empty galaxy space dimensions just doing nothing, the pack isn't gonna get better by adding mods arbitrarily, but people saying that a mod would be too laggy or have to much overlap to add, should really consider what mods the pack already has, i'm guessing no one wants to try to argue this over save compatibility and how much GT compat/tweaks was done for it, but there's a real argument that aether by it self adds more real content than galaxyspace, and would be better on servers

**Comment by karikamiya** [comment]
> GPLv3 Licensed

I'd just like to add that only the code itself is GPL, not the assets (which they define as anything in the /resources folder.

In typical Minecraft fashion, the website/github makes several assertions in the typical "OhCee doughnut steel" fashion, all incorrect. They seem to be the spergery type, so tread lightly. If they don't like your changes they might have a hissy fit even though the code is GPL

**Comment by Prometheus0000** [comment]
> I'd just like to add that only the code itself is GPL, not the assets (which they define as anything in the /resources folder.

Where does it specify there's a separate license for the assets?

**Comment by mitchej123** [comment]
> Where does it specify there's a separate license for the assets?

In the readme?

**Comment by karikamiya** [comment]
"The source code of The Aether mod for Minecraft 1.7+ is under the GNU GPL v3 license. All assets of the The Aether for Minecraft 1.7+ (found in the resources folder) are unlicensed and all rights are reserved to them by Gilded Games."

From readme.md

Also like to note that calling it "unlicensed" means nobody anywhere is allowed to use them and anyone using the mod is doing so illegally, and is probably in disagreement with any of the mod sites that they've uploaded the mod to. Typical "grr copyright" nonsense

**Comment by mitchej123** [comment]
It also looks like the 1.7.10 version is being actively maintained; I'd hesitate to fork it without talking to them first.

What sort of changes would we want to see implemented?  Would definitely be worth talking with them first.

**Comment by Dream-Master** [comment]
We need to add a lt of changes just for GTNH. I guess no one is willing to add it to main version

**Comment by mitchej123** [comment]
> We need to add a lt of changes just for GTNH. I guess no one is willing to add it to main version

_What_ changes.  Recipe changes? Easily done elsewhere. 

What specifically would we want to change that would require a fork?  Most mod authors that actively maintain mods don't enjoy forks for no good reason.

**Comment by Dream-Master** [comment]
not only recipes. I guess we change more. Monster hp points gt ores and stuff idk

**Comment by Vernam** [comment]
> not only recipes. I guess we change more. Monster hp points gt ores and stuff idk

i think there needed something that will motivate players to go to the Aether, some ores will not do this ( except this ores will be powerful). And there needed motivation to kill bosses. For TF there is ingredients for magic wands. Maybe it will be interesting if bosses will drop ingredients for Pure tear for warp cleaning without Luv tier - this is Very powerful motivation for all, especially for those who did not count on the warp and accidentally switched to serious debuffs. I write this as a idea for a future, gust suggestions =)

**Comment by bartimaeusnek** [comment]
> not only recipes. I guess we change more. Monster hp points gt ores and stuff idk
> 
> 

That's all stuff we can do in the core mod. It doesn't need a fork yet. If we discovered breaking bugs or incompatibilities, we can fork it, but currently there is no need for it.

**Comment by Alastors** [comment]
I'm reopening this, the aether does have some interesting usage in that we could use it specifically to add new content in general for earlier tiers, I don't believe it should be a hard gate at all but I'm sure there are some things we could do to give a reason to go to it

**Comment by vovaksenov** [comment]
Maybe first focus on finishing botania, new chem lines and polishing existing things? I genuinely believe the pack would benefit greatly from not adding anything new for a while, just to focus on making the existing things better. Maybe set adding aether as a goal for 2.3 or something?

**Comment by mitchej123** [comment]
Reopen it when it's closer to reality, we've got too much in flight, there's 0 chance of this happening for the next stable series...

**Comment by mitchej123** [comment]
Plus if I remember, the code might be open, but the assets were restricted, which means we wouldn't (easily) be able to distribute a modified version without special permissions.

**Comment by Alastors** [comment]
> Plus if I remember, the code might be open, but the assets were restricted, which means we wouldn't (easily) be able to distribute a modified version without special permissions.

ughhhh gross, I guess that alfheim fork someone's working on for the pack is a better alternative by far then?

**Comment by Alastors** [comment]
who's working on that anyway?

**Comment by Dream-Master** [comment]
i just load it in the pack but it was aether 2 i guess. If Botania is added and balanced 100% we can think of new mods.

**Comment by LunarLambda** [comment]
Has there been any interest in re-opening this RFC? People ask about Aether being added fairly frequently. Since Astral Sorcery has been deemed no-go due licensing issues maybe this could be an alternative? It adds a nice set of decorative blocks, has a Twilight Forest-ish boss progression, and could be an interesting way to gate certain magic-related resources behind some form of flight (Jetpack, etc.)

Twilight Forest really just exists to gate a few resources into LV and Thaum Wand Progression, I could see Aether playing a similar role for parts of Botania and certain Thaum materials. Plus I think there's a few niche items in the mod that might have useful interactions with the rest of the pack.

**Comment by mitchej123** [comment]
The assets are ARR.

**Comment by Cata-bre** [comment]
> The assets are ARR.

If new assets and textures are made, could we then consider adding it?
